### ISSUE TITLE
[SUGGESTION] Buying a protection for your drawing with tokens 

### ISSUE THREAD

**Issue by Shaydabop** [OurSources/owop-client#20]
If there were to ever be a token system in owop (which DayDun has talked about tokens before) we could buy a protection for a day. I think that it would be cool to have a daily login reward which is 10 tokens a day, and then people can spend their tokens on things. One of the things I suggest is temporary protection. I also think that temporary protection should be a little bit expensive, because protection is mainly a mod and admin feature. Here is how I would set up the price for temporary protection:
16 tokens per chunk (or 1 token per pixel if you plan on adding freely shaped protection)
Every day you would have to renew it by paying the price, but if you don't pay, your protection will get removed.

**Comment by SuperOP535** [comment]
what if ppl protect other ppl's art while they're drawing?

**Comment by DayDun** [comment]
Don't draw on unprotected land

**Comment by SuperOP535** [comment]
that would make it mandatory to have an account if you want to draw in public, maybe the main world should be user-protection-free?

**Comment by DayDun** [comment]
Accounts will probably be required to draw at all after the account update

**Comment by SuperOP535** [comment]
I hope it never comes then x)

**Comment by DayDun** [comment]
Why, do you have any better suggestions for dealing with proxy attacks?

**Comment by SuperOP535** [comment]
Because the player amount would probably lower quite a bit as it takes time to register if there is no option to link with Google etc. And I have no idea but having captcha always on to prevent attacks

**Comment by DayDun** [comment]
Don't assume thing with the accounts, you will log in with Google. And you can bypass captcha

**Comment by SuperOP535** [comment]
Sorry if I understand wrongly but by bypassing captcha do you mean with or without accounts?

**Comment by DayDun** [comment]
I was replying to your suggestion, captcha doesnt prevent bots. Bots can solve captcha

**Comment by SuperOP535** [comment]
Yes but it's not that easy, you may want to try something with cookies or keys
